images/rescue-image.rst

```diff
@@ -0,0 +1,27 @@
+============
+Rescue image
+============
+
+Sometimes you need to boot your instance from a ISO image to fix things that are not possible to fix when the filesystem on the boot :doc:`volume </storage/persistent-block-storage/index>` volume is mounted (that is; the instance is booted from it). To do this, you can use a rescue image.
+
+While there is a function in the :doc:`/getting-started/managing-your-cloud/cloud-management-portal`, it uses an old version of the underlying OpenStack functionality and is not recommended. 
```

Comment:
it does not support instance that is booting from a volume (where root disk is a volume)